internal/services/compute/windows_virtual_machine_scale_set_resource.go

```diff
@@ -1464,6 +1481,24 @@ func resourceWindowsVirtualMachineScaleSetSchema() map[string]*pluginsdk.Schema
 
 		"spot_restore": VirtualMachineScaleSetSpotRestorePolicySchema(),
 
+		// This field is Optional+Computed for:
+		// 1. Backward compatibility - existing scale sets won't show diffs when upgrading the provider
+		// The Computed attribute ensures Terraform reflects the actual Azure state.
+		"resilient_vm_creation_enabled": {
+			Type:     pluginsdk.TypeBool,
+			Optional: true,
+			Computed: true,
```

Comment:
Would you mind moving all the O+C notes between `Optional` and `Computed`? Per [the guide](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/best-practices.md#setting-properties-to-optional--computed) it will make them easier to find.
```suggestion
		"resilient_vm_creation_enabled": {
			Type:     pluginsdk.TypeBool,
			Optional: true,
			// NOTE: O+C ensures that scale sets that predate these properties won't show diffs
			Computed: true,
```